src/web/components/ClientSideTokenGeneration/CstgDomainsTable.tsx

```diff
@@ -95,6 +92,10 @@ export function CstgDomainsTable({
     );
     setShowDeleteDomainsDialog(false);
     setSelectedDomains([]);
+    setSearchedDomains([]);
+    setSearchText('');
+    if (deleteDomains.length === 1) setDomainAction('DeleteOne');
```

Comment:
nit: perhaps we should be consistent with how we format `if-else` blocks. Just below on line 102 through 106 we use brackets. Or we could use a ternary but doesn't bother me: 
```
setDomainAction(deleteDomains.length === 1 ? 'DeleteOne' : 'DeleteMany');
```